### PR TITLE
Synapse-Class Viral Study Vessel

### PR DESCRIPTION
**Pull request by ForestCranked**
## About The Pull Request

This is the Synapse-Class Viral Study Vessel, an NT (Specifically DeForest Medical) virology/chemistry vessel.

## Why It's Good For The Game

We don't really have the niche of virology on any ships yet, so this fills that empty niche. Enjoy, hopefully.

## Changelog
:cl:
add: The Synapse-Class Viral Study Vessel, and it's corresponding config.
fix: Accidentally dropped the json in configs and the shuttles folder, so i fixed that.
/:cl:

### CODE REVIEW

**Comment by ForestCranked** [comment]
also uh, don't mind the stuff from the enigma ruin. I made this branch from the master of my fork, which had the enigma ruin.

**Comment by Latentish** [comment]
Amogus.

Ok all in all, you really need to rethink the shape of this shuttle. As it stands this is mostly a rectangle with a half circle on top. (Yes I see the thruster rooms, but that really isn't enough of a deviation.)
The ship itself is supposed to be a viro and chem ship, but it seems like viro and chem are rather small and seem componential rather than integral.
Try expanding the viro and chem areas, give the viro area shutters to be able to lock down, and at least 2 individual patient rooms.
I'd also recommend making adding a small botany section with only 1 or 2 trays, so they can grow various plants for chemicals they can't usually get, or for cures they can't usually get.

**Comment by Apogee-dev** [comment]
I can't add much here that Sylv didn't already mention. The ship's shape is bland, the layout is pretty awkward (particularly on the starboard side of the ship), and it's mostly undecorated. I don't think you can really fix the issues with this one without fundamentally redesigning it.

**Comment by Latentish** [comment]
-Add 2 smaller rooms to the viro room in addition to the big room. Viro needs at least 2 isolation rooms for monkeys being infected, and a larger room is a good monkey pen. You could also replace the larger room with a grass pen.
-Make windows in the morgue into walls.
-Add a couple tables, chair, and supplies (formaldehyde, paper rack, pen, folders, body bag box, gloves box, facemask box, and cleaner are all good additions).

**Comment by Latentish** [comment]
image
please resolve this issue.

**Comment by ForestCranked** [comment]
i can't fix the pipe going from the tank since the gods hate me. both the init direction and normal direction should be facing towards the pipe, what's wrong?

**Comment by ForestCranked** [comment]
nevermind, got it working.

**Review by Latentish — CHANGES_REQUESTED** [review]
Please add a Cargo Console somewhere and we can TM this.

**Comment by Latentish** [comment]
I will be honest with you. I am not sure that virology on its own is enough to warrant it's own ship. That being said this ship fills its niche well, and has a rather unique shape. It's a decently pretty ship and you've followed most or all of our guidelines.  I'd like to get this fully closed, but your commits have no descriptions or titles , they all say "via upload". This makes it really difficult to see what changes you're making outside of what the picture in the description looks like at any given moment. Please start describing and titling your commits.
I'd like to ask if you've noticed anything playing it during test merge? The point of a test merge is to test your PR on live, so if you haven't played on your ship yet, I'd ask you do so next uptime, and let me know if you see anything.

I'd also like to ask the other @shiptest-ss13/maptainers what they think about the ship.

**Review by triplezeta — APPROVED** [review]
the lighter is good :)

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -805,6 +805,13 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		. = "<span class='rose'>[user] spits fire at [A], igniting it.</span>"
 		playsound(src, 'sound/magic/fireball.ogg', 10, TRUE)
 
+/obj/item/lighter/enigma
+	name = "Enigma Shipworks Lighter"
```
Comment (triplezeta) [inline code/game/objects/items/cigs_lighters.dm]:
```suggestion
	name = "\improper Enigma Shipworks Lighter"
```

**Review by Apogee-dev — APPROVED** [review]
Like Sylv, I have severe concerns about the viability of a viro-focused ship and I'm hesitant to approve this for merging. Nevertheless you've met all of our guidelines and to my knowledge no issues have arisen during testmerge. In fact it seems to have gained at least a handful of dedicated players. We'll have to see how it pans out in the long term.

**Review by Latentish — APPROVED** [review]

**Comment by Latentish** [comment]
Heres the deal. We don't really know if we want a ship to have viro. That being said, it isn't an issue right now? So if we get to the point that we want that gone, we can easily retool this ship. Soooo marg.

**Pull request merged**